### PR TITLE
Setup tracked courses empty state screen

### PR DESCRIPTION
**Pull request by haiyingweng**
### Overview
Added the navigation bar and footer
Added the search bar
Added currently tracked courses label 

### Screenshot
Screen Shot 2021-03-08 at 3 48 15 PM

### CODE REVIEW

File: src/components/Search/SearchBar.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
+.search-container {
+  width: 583px;
```
Comment (mathewscullin) [inline src/components/Search/SearchBar.css:2]: Should this be some `%` or `vw` rather than a static `px` value?

**Review by mathewscullin — APPROVED** [review]
Great work! 🥇

File: src/components/Footer/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+import './Footer.css';
+
+class Footer extends React.Component {
+  render() {
+    return (
+      <div className="footer">
+        <div className="container">
+          <p>
+            Originally created by Ning Ning Sun and Chase Thomas in 2016. Currently maintained by <a href="https://cornellappdev.com">Cornell AppDev.</a>
```
Comment (danielVebman) [inline src/components/Footer/Footer.tsx:11]: The nitiest pick ever, but I would put the period outside of the link.

**Review by danielVebman — APPROVED** [review]
Amazing work!

File: src/components/Search/SearchBar.css (hunk 1 above)
Comment (haiyingweng) [reply src/components/Search/SearchBar.css:2]: yes haha i was just copying the width from figma, but using % or vw would def be better especially if thinking about responsiveness

**Pull request merged**